src/Checker/SchemaChecker.php

```diff
@@ -45,8 +45,10 @@ public function check(string $schemaPath): array
                     if (empty($responseConfig->examples) || empty($responseConfig->examples->{'application/json'})) {
                         $summary[$path][$method][$response] = [
                             'errors' => [
-                                'path' => '',
-                                'error' => 'No example given',
+                                [
+                                    'property' => '',
+                                    'error' => 'No example given',
+                                ],
```

Comment:
there was a bug in the checker when response schema did not contain examples :hand_over_mouth: